lib/rubocop/cop/generator.rb

```diff
@@ -71,11 +71,17 @@ class %<cop_name>s < Base
                   (send nil? :bad_method ...)
                 PATTERN
 
+                # Called on every `send` node (method call) while walking the AST.
+                # TODO: remove this method if inspecting `send` nodes is unneeded for your cop.
+                # By default, this is aliased to `on_csend` as well to handle method calls
+                # with safe navigation, remove the alias if this is unnecessary.
+                # If kept, ensure your tests cover safe navigation as well!#{'  '}
```

Comment:
What's with the `#{}` here?

Reply:
It's `Layout/TrailingWhitespace` being over-zealous in a heredoc and I missed it. Updated now, thanks!